Realm/RLMSyncUtil.mm

```diff
@@ -19,6 +19,25 @@
 #import <Foundation/Foundation.h>
 
 #import "RLMSyncUtil_Private.hpp"
+#import "RLMSyncUser_Private.hpp"
+#import "RLMRealmConfiguration+Sync.h"
+#import "RLMRealmConfiguration_Private.hpp"
+#import "RLMSyncPermissionChange.h"
+
+@implementation RLMRealmConfiguration (RealmSync)
++ (instancetype)configurationForUser:(RLMSyncUser *)user {
```

Comment:
The name of this method should indicate that this will return a configuration for a management-Realm, given that there will be other special purpose user-specific Realms.